phpstan.neon.dist

```diff
@@ -1,10 +1,18 @@
 parameters:
     level: 5
+    bootstrapFiles:
+        - vendor/bin/.phpunit/phpunit/vendor/autoload.php
     paths:
         - src
-    excludes_analyse:
-        # temporarily ignore template files
-        - src/Resources/**.tpl.php
+        - tests
+    excludePaths:
+        analyseAndScan:
+            # temporarily ignore template files
+            - src/Resources/**.tpl.php
+            # BC-layer is not supported by phpstan https://github.com/phpstan/phpstan/issues/3542
+            - tests/Fixtures/StubTranslator.php
+    ignoreErrors:
+        - '#Trying to mock an undefined method [a-zA-Z]*\(\) on class stdClass\.#'
```

Comment:
Maybe we should add the phpstan baseline, just like with psalm?

Reply:
Done